microservices/content-management/model/Course.js

```diff
@@ -5,8 +5,8 @@ class Course {
         this.id = id;
         this.title = title;
         this.description = description;
-        this.creator = creator;
-        this.category = category;
+        this.categories = categories;
+        this.teachers = teachers;
```

Comment:
Course has only one teacher. Alias this teacher as creator. See my entities for reference

Reply:
I've that the people use their relation's names

Reply:
Yes but in the db the column is creator_id so it also could be creator but basically it is the same

Reply:
Will be fixed in the next pull request.